pages/shared/studentGroupAssessmentHelpers/studentGroupAssessmentHelpers.ejs

```diff
@@ -0,0 +1,43 @@
+<% if (groupsize == 0) { %>
```

Comment:
This file should be moved to `pages/partials` and named something different - perhaps `studentGroupControls`?

Reply:
Done in [8652798](https://github.com/PrairieLearn/PrairieLearn/pull/5290/commits/86527983f782382ca918fb20f3c9b995e3862239)